content/millennium/r4/scheduling/slot.md

```diff
@@ -193,6 +193,63 @@ List an individual Slot by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Patch
+
+Patch an existing Slot.
+
+    PATCH /Slot/:id
+
+_Implementation Notes_
+
+* This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
+* Only operations on the paths listed below are supported.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Accept': 'application/fhir+json',
+                   'Content-Type': 'application/json-patch+json', 'If-Match': 'W/"&lt;Current version of the Slot resource>"'} %>
+
+
+### Patch Operations
+
+<%= patch_definition_table(:slot_patch, :r4) %>
+
+### Example
+
+#### Request
+
+    PATCH https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Slot/4047611-32216049-61518614-0
```

Comment:
This should be `https://fhir-ehr-code.cerner.com/r4/`

Reply:
done

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/436435c75110e6b389fd06d812050eb025b4cfca#diff-3251e7cb3d59e4aa73da18f3e0aa8a09553c8abaf90868d2811ccfced1e09d0bR230)